sys-boot/tboot/tboot-1.10.3.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit mount-boot toolchain-funcs
+
+DESCRIPTION="Performs a measured and verified boot using Intel Trusted Execution Technology"
+HOMEPAGE="https://sourceforge.net/projects/tboot/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="custom-cflags selinux"
+
+# requires patching the kernel src
+RESTRICT="test"
+
+DEPEND="|| ( >=dev-libs/openssl-1.1.1l-r1
+	<dev-libs/openssl-1.1.1l-r1[-bindist] )"
+
+RDEPEND="${DEPEND}
+	sys-boot/grub:2
+	selinux? ( sec-policy/selinux-tboot )"
+
+DOCS=( README.md COPYING CHANGELOG )
+PATCHES=( "${FILESDIR}/${PN}-1.9.11-genkernel-path.patch" )
+
+src_prepare() {
+	sed -i 's/ -Werror//g' Config.mk || die
```

Comment:
Can we use a patch for these `sed`s instead?

Reply:
I turned 2 of them (the ones to remove stripping and -Werror) into patches. I combined 10 them into 1, leaving 2 left. The cc issue should be fixed now too.